We don't have all dependencies required for `jsonschema[format-nongpl]`.
However, `rfc3339-validator` and `rfc3986-validator` seem enough, and
we do have them listed in `depends=`.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@
 ]
 dependencies = [
     "referencing",
-    "jsonschema[format-nongpl]>=4.18.0",
+    "jsonschema>=4.18.0",
     "python-json-logger>=2.0.4",
     "pyyaml>=5.3",
     "traitlets>=5.3",
